### PR TITLE
[Coral] Bug fix for the DB connection lifetime

### PR DESCRIPTION
**Pull request by qliphy**
backport of cms-sw/cmsdist#6771
meant for next MWGR

### CODE REVIEW

**Comment by qliphy** [comment]
please test

File: coral.spec [hunk 1]
```diff
@@ -1,43 +1,4 @@
...
-# Drop Oracle interface on ARM machines and POWER machines.
-# Oracle does not provide Instant Client for ARMv8 or POWER8.
-%if 0%{?cmsplatf_aarch64}%{?cmsplatf_ppc64le}%{?cmsplatf_ppc64}
-%define patchsrc2       rm -rf ./src/OracleAccess
-%endif
-%define patchsrc4       %patch1 -p1
-
-%define source1  git://github.com/%{github_user}/%{n}.git?protocol=https&obj=%{branch}/%{tag}&module=%{cvssrc}&export=%{srctree}&output=/src.tar.gz
-## IMPORT scram-project-build
-## SUBPACKAGE debug IF %subpackageDebug
```
Comment (smuzaffar) [inline coral.spec]: this is wrong, we should not delete all of the above

Comment (qliphy) [reply coral.spec]: Thanks. Fixed now.

**Comment by qliphy** [comment]
please test

**Comment by smuzaffar** [comment]
+externals
looks good, input relval tests failed due to das/dbs glitches

**Comment by smuzaffar** [comment]
@qliphy , feel free to merge it

**Comment by qliphy** [comment]
+1

**Comment by qliphy** [comment]
merge

**Comment by davidlange6** [comment]
we are 100% sure this doesn’t need any relval testing?

**Comment by smuzaffar** [comment]
@davidlange6 , do you mean PR tests or proper validation? The PR tests ran fine 
- https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-13c4cc/13876/runTheMatrixINPUT-results/runall-report-step123-.log
  - 595 595 592 553 102 41 3 1 1 tests passed, 0 0 0 0 0 0 0 0 0 failed
  - few Input relvals (where we try to run all workflows with inputs) failed due to dasgoclient glicthes like
```
2021/03/31 12:30:27 ERROR failed to parse X509 proxy: crypto/tls: failed to parse key: asn1: syntax error: sequence truncated
2021/03/31 12:30:27 ERROR failed to parse X509 proxy: crypto/tls: failed to parse key: asn1: syntax error: sequence truncated
``` 
- https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-13c4cc/13876/runTheMatrix-results/runall-report-step123-.log
  - 36 35 34 25 16 4 1 1 1 tests passed, 0 0 0 0 0 0 0 0 0 failed

**Comment by davidlange6** [comment]
Proper validation (usually required for coral changes) - but perhaps its not required in this case.

**Pull request merged**